### ISSUE TITLE
disable search in top menu

### ISSUE THREAD

**Issue by LarsRabe** [JustCarmen/webtrees-theme-justlight#139]
To disable the search header I added the following code to the `ccs and js` module:

`<style>
.wt-header-search { display: none; }
</style>`

This works with the standard webtrees theme but not with your justLight or justBlack theme (see sreenshots). Is this a bug or intentional? Can this be changed?
![zoeken_1](https://user-images.githubusercontent.com/119816325/215327357-710c3b48-6f94-47a0-aae1-ec2346389024.jpg)
![zoeken_2](https://user-images.githubusercontent.com/119816325/215327363-b7c3684c-d5fc-44af-86c9-71160e7c3dfc.jpg)

**Comment by JustCarmen** [comment]
This is the result the navbar-collapse class on the same element which is necessary for small screens.

Use `display: none !important` to achieve what you want.

**Comment by LarsRabe** [comment]
I changed it to 
`<style> .wt-header-search { display: none !important; } </style>`
as you suggested.

Sadly that doesn't work either. Any other suggestion?

**Comment by JustCarmen** [comment]
I've made some changes to the code. Now it should work with your original custom css (`<style> .wt-header-search { display: none; } </style>`)

**Comment by LarsRabe** [comment]
I just found time to test. Sadly thes changes don't work either.

But I achieved what I wanted by 
`.wt-header-search-form { display: none; }`
This works on all themes including yours.

So somewhere in your code the `.wt-header-search` still seems to behave different from the original themes (I've tested all the standard themes).

**Comment by JustCarmen** [comment]
> So somewhere in your code the .wt-header-search still seems to behave different from the original themes 

Yes, that's because the Justlight theme is optimized for mobile use and the default themes are not. I use a special mobile menu with the bootstrap navbar-collapse class. This conflicts with your custom changes because they both require the css display function on the same element.

**Comment by LarsRabe** [comment]
>But '<style> .wt-header-search { display: none; }' </style>' works in my local test system, so I will leave the changes as they are.
I've tried it again. I made sure not to have any typo, deleted your theme and reinstalled with all of your development patches. Even disabled all other custom modules. Tried another computer and tablet. But, no luck at all. So it seems my server behaves different from your local test system. I can only get it to work by the `-form`class.

Could it be possible it's PHP version related? I'm PHP 8.2.1
